packages/dashboard/src/components/pages/about-us/about-us.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { Text } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import Frame from 'Components/elements/frame';
+import { useStores } from 'Stores';
+
+const AboutUs: React.FC = () => {
+    const { config_store } = useStores();
+    const total_previews = 27;
+
+    return (
+        <div className='dw-about-us'>
+            <Text className='dw-about-us__text dw-about-us__title' as='h1' size='l' color='general' align='center'>
```

Comment:
Do we have support for headings in Text? (`as='h1'`)

Reply:
This is a temporary page so let's not worry about styling.